src/main/java/seedu/address/model/person/Tissuetype.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's tissue type in ORGANice.
+ * Guarantees: immutable; is valid as declared in {@link #isValidTissuetype(String)}
+ */
+public class Tissuetype {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Tissue types are 6 integers from 1 to 12 separated by commas"
+                + " An example will be tt/1,2,3,4,5,6";
+    public final String value;
+
+    /**
+     * Constructs a {@code Tissuetype}.
+     *
+     * @param tissuetype A valid tissue type
+     */
+    public Tissuetype (String tissuetype) {
+        requireNonNull(tissuetype);
+        checkArgument(isValidTissuetype(tissuetype), MESSAGE_CONSTRAINTS);
+        value = tissuetype.toUpperCase();
+    }
+
+    /**
+     * Returns true if a given string is a valid tissue type.
+     */
+    public static boolean isValidTissuetype(String test) {
+        String[] tissuetypeValue = test.split(",");
+        if (tissuetypeValue.length != 6) {
+            return false;
+        } else {
+            for (int i = 0; i < tissuetypeValue.length; i++) {
+                try {
+                    Integer tt = Integer.parseInt(tissuetypeValue[i]);
```

Comment:
There is no check for whether the tissue types are numbers from 1-12

Reply:
can you add check that all numbers is non duplication and from range 1-12.

Reply:
Actually according to the link jiaying gave and google, there seems to be more than 12 types of tissue. But I cant really find a fixed number online on how manu types exactly. Should we just assume its 12?

Reply:
I did some additional research. Apparently there are a lot more tissue types. We can stick to 12 tissues.

Reply:
Ok noted. Will change by tonight. I can make to 20 actually since I think the total type is more than 20

Reply:
just stick with 12 for simplicity. Thanks!

Reply:
so just have a check if all number is <= 12 and there is non duplicate

Reply:
so just have a check if all number is <= 12 and there is non duplicate

Reply:
Can add comment that its > 12 types but we are limiting it for now. Perhaps add `//TODO: comment`

Reply:
Ok noted